Public/Src/Tools/Execution.Analyzer/Analyzers.Core/XLGPlusPlus/XLGppDataStore.cs

```diff
@@ -0,0 +1,111 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System;
+using System.Collections.Generic;
+using System.Diagnostics.ContractsLight;
+using BuildXL.Engine.Cache.KeyValueStores;
+using BuildXL.Execution.Analyzer;
+using BuildXL.Scheduler.Tracing;
+using BuildXL.Utilities;
+using Google.Protobuf;
+
+namespace BuildXL.Analyzers.Core.XLGPlusPlus
+{
+    public class XLGppDataStore
+    {
+        /// <summary>
+        /// Rocks DB Accessor for XLG++ data
+        /// </summary>
+        public KeyValueStoreAccessor Accessor { get; set; }

```

Comment:
>KeyValueStoreAccessor [](start = 15, length = 21)

this thing implements IDisposable, should you implement it as well? Check what it's doing/how we are using it at other places.

Reply:
Done, made it implement IDisposable, and also implement the Dispose() function

---
In reply to: [306050332](https://github.com/microsoft/BuildXL/pull/636#discussion_r306050332) [](ancestors = 306050332)